python/dgl/graphbolt/minibatch.py

```diff
@@ -562,29 +562,7 @@ def _to(x, device):
         def apply_to(x, device):
             return recursive_apply(x, lambda x: _to(x, device))
 
-        if self.seed_nodes is not None and self.compacted_node_pairs is None:
-            # Node related tasks.
-            transfer_attrs = [
-                "labels",
-                "sampled_subgraphs",
-                "node_features",
-                "edge_features",
-            ]
-            if self.labels is None:
-                # Layerwise inference
-                transfer_attrs.append("seed_nodes")
-        elif self.seed_nodes is None and self.compacted_node_pairs is not None:
-            # Link/edge related tasks.
-            transfer_attrs = [
-                "labels",
-                "compacted_node_pairs",
-                "compacted_negative_srcs",
-                "compacted_negative_dsts",
-                "sampled_subgraphs",
-                "node_features",
-                "edge_features",
-            ]
-        elif self.seeds is not None:
+        if self.seeds is not None:
```

Comment:
Do we still need this check?

Reply:
> Do we still need this check?

Without this check, `input_nodes` will be transfer to the device as well, is this acceptable?